rest-dashboard.go

```diff
@@ -116,6 +116,38 @@ func (r *Client) GetDashboardBySlug(ctx context.Context, slug string) (Board, Bo
 	return r.getDashboard(ctx, path)
 }
 
+// DashboardVersion represents a response from /api/dashboards/id/:dashboardId/versions API
+type DashboardVersion struct {
+	ID            uint      `json:"id"`
+	DashboardID   uint      `json:"dashboardId"`
+	ParentVersion uint      `json:"parentVersion"`
+	RestoredFrom  uint      `json:"restoredFrom"`
+	Version       uint      `json:"version"`
+	Created       time.Time `json:"created"`
+	CreatedBy     string    `json:"createdBy"`
+	Message       string    `json:"message"`
+}
+
+// GetDashboardVersionsByDashboardID reflects /api/dashboards/id/:dashboardId/versions API call
+func (r *Client) GetDashboardVersionsByDashboardID(ctx context.Context, id uint, params ...QueryParam) ([]DashboardVersion, error) {
```

Comment:
Could we please rename this variable so that it would reflect whose ID it is? It appears in code suggestions so I think it is important to have a descriptive name here.

```suggestion
func (r *Client) GetDashboardVersionsByDashboardID(ctx context.Context, dashboardID uint, params ...QueryParam) ([]DashboardVersion, error) {
```

Reply:
I do agree. I renamed the var.